mypy/typetraverser.py

```diff
@@ -86,6 +86,9 @@ def visit_callable_type(self, t: CallableType) -> None:
         t.ret_type.accept(self)
         t.fallback.accept(self)
 
+        if t.type_guard is not None:
+            t.type_guard.accept(self)
```

Comment:
Is this an issue for `TypeIs` as well? Probably as easy as adding an `if t.type_is is not None` clause here.

Reply:
Yes, please add an analogous test for `TypeIs` as well. It should behave exactly the same here.

Reply:
Done, thanks! It was interesting to learn about `TypeIs` :)